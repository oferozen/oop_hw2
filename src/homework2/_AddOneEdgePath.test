# Add an additional path that's comprised of one edge

# Create a graph
CreateGraph graph1

# Add three nodes to graph1
CreateNode n1 1
CreateNode n2 2
CreateNode n3 3
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Add the directed edge (n1,n2)
AddEdge graph1 n1 n2

# Add the directed edge (n2,n3)
AddEdge graph1 n2 n3

# Add the directed edge (n1,n3)
AddEdge graph1 n1 n3

# Graph status
ListNodes graph1
ListChildren graph1 n1
ListChildren graph1 n2
ListChildren graph1 n3