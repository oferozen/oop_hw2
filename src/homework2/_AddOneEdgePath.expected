# Add an additional path that's comprised of one edge

# Create a graph
created graph graph1

# Add three nodes to graph1
created node n1 with cost 1
created node n2 with cost 2
created node n3 with cost 3
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Add the directed edge (n1,n2)
added edge from n1 to n2 in graph1

# Add the directed edge (n2,n3)
added edge from n2 to n3 in graph1

# Add the directed edge (n1,n3)
added edge from n1 to n3 in graph1

# Graph status
graph1 contains: n1 n2 n3
the children of n1 in graph1 are: n2 n3
the children of n2 in graph1 are: n3
the children of n3 in graph1 are:
